modules/webhooks.py

```diff
@@ -24,7 +29,30 @@ def __init__(self, config, system_webhooks, notifiarr=None, apprise=None):
         self.notifiarr = notifiarr
         self.apprise = apprise
 
+    def request_and_check(self, webhook_type, json):
```

Comment:
Can we rename the `webhook_type` parameter to `webhook` ? Currently if we keep it as `webhook_type` it gets overwritten in line 46.